client/OPUNetGameSelectWnd.cpp

```diff
@@ -589,13 +589,13 @@ bool OPUNetGameSelectWnd::OnReceiveJoin(Packet& packet)
 	// Make sure we've requested to join a game
 	if (joiningGame == nullptr)
 	{
-		SetStatusText("Unexpected Join reply received");
+		LogDebug("Join reply received with wrong Session ID");

```

Comment:
Copy/paste error? The text was changed here. It no longer matches the check that was being done.